error/src/lib.rs

```diff
@@ -578,10 +578,14 @@ impl CaliptraError {
     pub const KAT_HMAC384_TAG_MISMATCH: CaliptraError = CaliptraError::new_const(0x90030002);
 
     pub const KAT_ECC384_SIGNATURE_GENERATE_FAILURE: CaliptraError =
-        CaliptraError::new_const(0x90040001);
+        CaliptraError::new_const(0x90040001); // Deprecated
```

Comment:
You might move these to `///` comments above so that it shows up nicer in an IDE or documentation.